lldb/source/Interpreter/CommandInterpreter.cpp

```diff
@@ -1160,7 +1160,9 @@ Status CommandInterpreter::AddUserCommand(llvm::StringRef name,
 
   if (UserCommandExists(name)) {
     if (!can_replace) {
-      result.SetErrorString("user command exists and force replace not set");
+      result.SetErrorString(
+          "user command exists and force replace not set by --overwrite or "
```

Comment:
Can we include the name of the command and "already" here, for example:
```
user command "foo" already exists and force replace not....
```

Reply:
Also "was not set by" ? (adding "was")

Reply:
Yup, good suggestions, I'll update it!